app.rb

```diff
@@ -0,0 +1,120 @@
+require './book'
+require './classroom'
+require './per_son'
+require './rental'
+require './student'
+require './teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def all_booklist
+    puts 'There is no book found in the store' if
+    @books.empty?
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def all_peoplelist
+    puts 'No people found!' if @people.empty?
+    @people.each { |person| puts "[#{person.class}], Name: #{person.name}, Age: #{person.age}, Id: #{person.id}" }
+  end
```

Comment:
- [ ] I am able to create a `student` or `teacher` but when I want to see the list I am only seeing teachers, students are not found please address that